### PR TITLE
E2E - Trade Component 

### PR DESCRIPTION
**Pull request by nalancer08**
### Type of change

- [ ] Chore
- [X] Feature
- [ ] Bug fix

### Why

Add E2E testing for Trade Component with buy/sell flows and deposit always 50 USD

### CODE REVIEW

File: CybridSDKTestAppUITests/Flow/TradeFlow.swift [hunk 1]
```diff
@@ -0,0 +1,822 @@
...
+        
+        // -- Given
+        let demoMode = app.buttons["demo_mode"]
+        demoMode.tap()
+    }
+    
+    func depositFounds() {
+        
+        // --
+        let foundsToAdd = "50"
```
Comment (brent-cybrid) [inline CybridSDKTestAppUITests/Flow/TradeFlow.swift]: `founds` should be `funds` (please update everywhere in the script)

File: CybridSDKTestAppUITests/Flow/TradeFlow.swift [hunk 2]
```diff
@@ -0,0 +1,822 @@
...
+            self.returnTap()
+            
+        } else {
+            
+            // -- Return main controller
+            self.returnTap()
+        }
+    }
+    
+    func trade_buy_SOL() {
```
Comment (brent-cybrid) [inline CybridSDKTestAppUITests/Flow/TradeFlow.swift]: For these buy/ sell flows, can we create a helper function to encapsulate all the logic and then just call the helper with the correct asset and action (i.e., Buy or Sell)?

File: CybridSDKTestAppUITests/Flow/TradeFlow.swift (hunk 1 above)
Comment (nalancer08) [reply CybridSDKTestAppUITests/Flow/TradeFlow.swift]: Done!

File: CybridSDKTestAppUITests/Flow/TradeFlow.swift (hunk 2 above)
Comment (nalancer08) [reply CybridSDKTestAppUITests/Flow/TradeFlow.swift]: Done!

**Review by brent-cybrid — APPROVED** [review]

**Pull request merged**